django/scalingo.json

```
@@ -1,7 +1,21 @@
 {
     "env": {
-        "ALLOWED_HOSTS": {
+        "APP_URL": {
             "generator": "url"
+        },
+        "ALLOWED_HOSTS": {
+            "generator": "template",
+            "template": "docurba-django-demo-pr%PR_NUMBER%.osc-fr1.scalingo.io"
```

Totalement optionnel :

```suggestion
            "template": "%APP%.osc-fr1.scalingo.io"
```